icu4c/source/i18n/unicode/numsys.h

```diff
@@ -73,6 +73,12 @@ class U_I18N_API NumberingSystem : public UObject {
      */
     NumberingSystem(const NumberingSystem& other);
 
+    /**
+     * Copy assignment (defaulted).
+     * @draft ICU 67
```

Comment:
What version number?

Reply:
same, these were default pre-existing. So i think it should be exactly the version and stability of the copy c'tor.

Reply:
so stable 4.2